contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -25,12 +27,30 @@ contract LPOptimiserStrategy is DefaultAccessControl, ILpCallback {
     IVoltzVault internal _vault;
     uint256[] internal _pullExistentials;
     IMarginEngine internal _marginEngine;
+    IPeriphery internal _periphery;
+    IVAMM internal _vamm;
     IVoltzVault.TickRange internal _currentPosition;
```

Comment:
I don't see why we need to keep track of _currentPosition here. We should retrieve it directly from the vault.

Reply:
Removed